dispatcher.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, task_groups, max_workers_cnt, randomize):
             self.result_queues.append(task_queue_disp.result_queue)
             self.task_queues.append(task_queue_disp.task_queue)
 
-        self.report_timeout = 1.0
+        self.report_timeout = 0.1
```

Comment:
It was in the 'sampler: add simple sampling infrastructure' commit and it would be good to leave it here. Otherwise the following statement in the commit message will lie:

> Add a function that will be called each 0.1 second for each tracked
process.